Sources/SwiftyRequest/RestRequest.swift

```diff
@@ -738,6 +740,9 @@ public enum Result<T> {
 public enum Credentials {
     /// an API key is being used, no additional data needed
     case apiKey
+  
+    /// a bearer authentication (token)
```

Comment:
Can you note here that the String should already be base64-encoded please?

Reply:
Yup, will do